spring-integration-http/src/test/java/org/springframework/integration/http/inbound/HttpRequestHandlingControllerTests.java

```diff
@@ -349,26 +349,24 @@ protected Object handleRequestMessage(Message<?> requestMessage) {
 		MockHttpServletResponse response = new MockHttpServletResponse();
 		final AtomicInteger active = new AtomicInteger();
 		final AtomicBoolean expected503 = new AtomicBoolean();
-		Executors.newSingleThreadExecutor().execute(new Runnable() {
-			public void run() {
-				try {
-					// wait for the active thread
-					latch2.await(10, TimeUnit.SECONDS);
-				}
-				catch (InterruptedException e1) {
-					Thread.currentThread().interrupt();
-				}
-				// start the shutdown
-				active.set(controller.beforeShutdown());
-				try {
-					MockHttpServletResponse response = new MockHttpServletResponse();
-					controller.handleRequest(request, response);
-					expected503.set(response.getStatus() == HttpStatus.SERVICE_UNAVAILABLE.value());
-					latch1.countDown();
-				}
-				catch (Exception e) {
-					e.printStackTrace();
-				}
+		Executors.newSingleThreadExecutor().execute(() -> {
+			try {
+				// wait for the active thread
+				latch2.await(10, TimeUnit.SECONDS);
+			}
+			catch (InterruptedException e1) {
+				Thread.currentThread().interrupt();
+			}
+			// start the shutdown
+			active.set(controller.beforeShutdown());
+			try {
+				MockHttpServletResponse response1 = new MockHttpServletResponse();
+				controller.handleRequest(request, response1);
+				expected503.set(response1.getStatus() == HttpStatus.SERVICE_UNAVAILABLE.value());
+				latch1.countDown();
+			}
+			catch (Exception e) {
+				e.printStackTrace();
```

Comment:
I don't like this `printStackTrace()`.
Right, existing code, but still.
Would you mind fixing since we are already here?
Thanks